lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/FileListCacheValue.java

```diff
@@ -165,7 +159,7 @@ public boolean equals(Object obj) {
    public String toString() {
       readLock.lock();
       try {
-         return "FileListCacheValue{fileNames=" + fileNames + "}";
+         return "FileListCacheValue{fileNames=" + getFileNames() + "}";
```

Comment:
This change I requested was supposed to remove the need for a read lock

Reply:
Argh yes, obviously I made the suggested change without engaging my brain. Updated!